libs/designsystem/src/lib/components/segmented-control/segmented-control.component.ts

```diff
@@ -78,6 +79,8 @@ export class SegmentedControlComponent {
     return this._value;
   }
 
+  @Output() selectedIndexChange = new EventEmitter<number>();
```

Comment:
Just out of curiosity, why is this new output needed?

We should remember to document it in the cookbook too! 😄

Reply:
Will do... - still in progress...
The selected index is both manipulated internally in response to mouse mouse clicks and programmatically. 
Defining this as a one-way binding means it doesn't get updated externally when it is changed internally (get out-of-sync) which in turn means that trying setting this value externally/programmatically afterwards may be ignored.

Reply:
Done..